README.md

```diff
@@ -79,7 +79,9 @@ Authors
 -------
 
 [Jamie Talbot](https://github.com/majelbstoat)
+
```

Comment:
Right now they render all on the same line. This change is so each one renders on its own line.